### PR TITLE
Minor imbd dataset documentation update, added docstring to reuters

### PR DESCRIPTION
**Pull request by reidsanders**
Changed the imbd dataset documentation to reflect all current options - most importantly the removal of the test_split option. I also included other options (oov_char, etc), though they are rather niche, so perhaps their lack of inclusion was intentional. I also added a docstring to reuters load_data for clarity.

### CODE REVIEW

File: docs/templates/datasets.md [hunk 1]
```diff
@@ -56,8 +56,7 @@ from keras.datasets import imdb
 (X_train, y_train), (X_test, y_test) = imdb.load_data(path="imdb.pkl",
                                                       nb_words=None,
                                                       skip_top=0,
-                                                      maxlen=None,
-                                                      test_split=0.1)
+                                                      maxlen=None)
```
Comment (fchollet) [inline docs/templates/datasets.md]: The 3 last keyword arguments should appear here (mostly to make the default values explicit).

File: docs/templates/datasets.md [hunk 2]
```diff
@@ -91,7 +94,9 @@ from keras.datasets import reuters
                                                          test_split=0.1)
```
Comment (fchollet) [inline docs/templates/datasets.md]: Same here.

**Comment by reidsanders** [comment]
I added all current arguments with their default values to the imbd and reuters usage examples.

**Comment by fchollet** [comment]
LGTM

**Pull request merged**